reference.md

```diff
@@ -2,39 +2,29 @@
 
 [Usage Tutorial](index)
 
-### robot.Camera
-Class that represents a camera.
-- **Methods**
-  - `see()`: returns a list of [markers](#robotmarker) currently visible to the camera sorted by distance (closest first).
-
-
-- **Properties**
-  - `serial`: the camera's serial number as a string.
-
 ---
-### robot.CartCoord
-Class that represents a set of cartesian coordinates.
-- **Properties**
-  - `x`: x-axis cartesian coordinate in metres.
-  - `y`: y-axis cartesian coordinate in metres.
-  - `z`: z-axis cartesian coordinate in metres.
+### robot.PinMode
+Holds constant values to represent the mode of a pin
+- **Values**
+  - `INPUT`: Make the pin be an input.
+  - `INPUT_PULLUP`: Make the pin be an input, with a pullup resistor.
+  - `OUTPUT_HIGH`: Set the pin to output high.
+  - `OUTPUT_LOW`: Set the pin to output low.
 
 ---
-### robot.Marker
-Class that represents a marker.
-- **Methods**
-  - `is_token_marker()`: if the marker is a token marker.
-  - `is_wall_marker()`: if the marker is a wall marker.
-
+### robot.PinValue
+Holds constant the values of a pin
+- **Values**
+  - `HIGH`: The pin is receiving a high current
+  - `LOW`: The pin is receiving a low current
```

Comment:
Please change "current" to "voltage"

Maybe also give examples of a "high" and "low" voltage? i.e. 5V and 0V respectively